commands/addvertical.js

```diff
@@ -119,12 +120,71 @@ class VerticalAdder {
    * @param {Object<string, string>} args The arguments, keyed by name 
    */
   execute(args) {
-    if (!VerticalAdder._getAvailableCards(this.config).includes(args.cardName)) {
+    this._validateArgs(args);
+    this._createVerticalPage(args.name, args.template);
+    const cardName = args.cardName || this._getCardDefault(args.template);
+    this._configureVerticalPage(args.name, args.verticalKey, cardName);
+  }
+
+
+  /**
+   * Structural validation (missing required parameters, etc.) is handled by YArgs. This
+   * method provides an additional validation layer to ensure the provided template and
+   * cardName are valid. Any issue will result in a {@link UserError} being thrown.
+   * 
+   * @param {Object} args The command parameters.
+   */
+  _validateArgs(args) {
+    if (args.template === 'universal-standard') {
+      throw new UserError('A vertical cannot be initialized with the universal template');
+    }
+
+    const themeDir = this._getThemeDirectory(this.config);
+    const templateDir = path.join(themeDir, 'templates', args.template);
+    if (!fs.existsSync(templateDir)) {
```

Comment:
I remember Bryan discussing that in general, we should first check for folders in the top-level of the site, and then check inside the theme as a fallback. Do you think that we should do the same here as well?

Reply:
Hmm, that's not a use case that Product anticipated. But, it could happen! Someone could shadow the `templates` directory and we'd want to honor that here. If you don't mind, I can do that as a separate item/PR. The command in it's current state doesn't support this either.

Reply:
Sure, I think a separate item for that makes sense